templates/quickstart-database-for-atlassian-services.yaml

```diff
@@ -0,0 +1,182 @@
+      (-).
+    Description: S3 bucket name for the Quick Start assets. Quick Start bucket name
+      can include numbers, lowercase letters, uppercase letters, and hyphens (-).
+      It cannot start or end with a hyphen (-).
+    Type: String
+  QSS3KeyPrefix:
+    Default: 'quickstart-atlassian-services/'
+    AllowedPattern: ^[0-9a-zA-Z-/]*$
+    ConstraintDescription: Quick Start key prefix can include numbers, lowercase letters,
+      uppercase letters, hyphens (-), and forward slash (/).
+    Description: S3 key prefix for the Quick Start assets. Quick Start key prefix
+      can include numbers, lowercase letters, uppercase letters, hyphens (-), and
+      forward slash (/).
+    Type: String
+
+Conditions:
+    UseAurora:
+      !Equals [!Ref DatabaseImplementation, 'AWS Aurora PostgreSQL']
+    UsePostgres:
+      !Equals [!Ref DatabaseImplementation, 'PostgreSQL']
+    GovCloudCondition:
+      !Equals [!Ref 'AWS::Region', 'us-gov-west-1']
+    DBProvisionedIops:
+      !Equals [!Ref DBStorageType, io1]
+    UseDatabaseEncryption:
+      !Equals [!Ref DBStorageEncrypted, true]
+
+Resources:
+  AuroraDatabase:
+    Type: AWS::CloudFormation::Stack
+    Condition: UseAurora
+    Properties:
+      TemplateURL: 
+        !Sub
+          - https://${QSS3BucketName}.${QSS3Region}.amazonaws.com/${QSS3KeyPrefix}submodules/quickstart-atlassian-services/submodules/quickstart-amazon-aurora/templates/aurora_postgres.yaml
+          - QSS3Region: !If ["GovCloudCondition", "s3-us-gov-west-1", "s3"]
+      Parameters:
+        DBBackupRetentionPeriod: !Ref DBBackupRetentionPeriod
+        DBInstanceClass: !Ref DBInstanceClass
+        DBMasterUsername: postgres
+        DBMasterUserPassword: !Ref DBMasterUserPassword
+        DBMultiAZ: !Ref DBMultiAZ
+        DBPort: '5432'
+        DBAllocatedStorageEncrypted: !Ref DBStorageEncrypted
+        CustomDBSecurityGroup: !Ref DBSecurityGroup
+        Subnet1ID: !Select [0, !Split [ ",", !ImportValue ATL-PriNets]]
+        Subnet2ID: !Select [1, !Split [ ",", !ImportValue ATL-PriNets]]
+        # NB: The VPCID is not used by the aurora template but it will fail if it is not provided.
+        VPCID: !ImportValue ATL-VPCID
+
+  PostgresDatabase:
+    Type: AWS::CloudFormation::Stack
+    Condition: UsePostgres
+    Properties:
+      TemplateURL:
+        !Sub
+          - https://${QSS3BucketName}.${QSS3Region}.amazonaws.com/${QSS3KeyPrefix}submodules/quickstart-atlassian-services/templates/quickstart-postgres-for-atlassian-services.yaml
+          - QSS3Region: !If ["GovCloudCondition", "s3-us-gov-west-1", "s3"]
+      Parameters:
+        DBAutoMinorVersionUpgrade: !Ref DBAutoMinorVersionUpgrade
+        DBBackupRetentionPeriod: !Ref DBBackupRetentionPeriod
+        DBInstanceClass: !Ref DBInstanceClass
+        DBIops: !Ref DBIops
+        DBMasterUsername: postgres
+        DBMasterUserPassword: !Ref DBMasterUserPassword
+        DBMultiAZ: !Ref DBMultiAZ
+        DBPort: '5432'
```

Comment:
This doesn't need to be passed because sub-template sets default 5432

Reply:
Actually, this parameter is required by the bootstrap template. So if we are passing it down, we may as well use it.